content/blog/2022-04-04-develop-and-test-data-analytics-pipelines-on-localstack/index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Develop and Test Real-time Data Pipelines with LocalStack
+description: "LocalStack makes it easy to develop and test real-time data pipelines that are built with AWS services. Learn how to deploy Cloudwatch, Kinesis, Lambda, and external services using CDK on LocalStack."
+lead: "LocalStack makes it easy to develop and test real-time data pipelines that are built with AWS services. Learn how to deploy Cloudwatch, Kinesis, Lambda, and external services using CDK on LocalStack."
+date: 2022-04-04
+lastmod: 2022-04-04
+draft: false
+images: []
+contributors: ["Sam Watson"]
+tags: ["tutorial"]
+---
+
+{{< img src="teaser.png" >}}
```

Comment:
nit: We should probably still update the teaser image, which I think was intended for a different blog post.

Reply:
I removed the reference to the teaser for now.